conversions/binary_to_decimal.c

```diff
@@ -1,24 +1,113 @@
-/**
- * Modified 07/12/2017, Kyler Smith
- *
+/** 
+ * @file binary_to_decimal.c
+ * @brief Converts a binary number to a decimal one.
+ * @details
+ * A binary number is input , it is check to be binary
+ * then number is converted to a decimal number.
+ * Some tests are added too.
+ * Modified 07/12/2017
+ * Modified 2021
+ * @author Kyler Smith
+ * @author [lazy-dude] (https://github.com/lazy-dude)
  */
 
+// includes
+#include <assert.h>
+#include <stdbool.h>
+#include <stdint.h> 
 #include <stdio.h>
+#include <stdlib.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>    /// for IO operations
#include <assert.h>  /// for assert
```